template_feed/Microsoft.DotNet.Test.ProjectTemplates.1.x/content/MSTest-VisualBasic/Company.TestProject1.vbproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' == ''">netcoreapp2.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
+
+    <IsPackable Condition="'$(EnablePack)' == 'true'">true</IsPackable>

```

Comment:
We still need the runner. (Not sure yet how we're getting that in the project system repo but it some msbuild magic)